web/ui/dashboard/src/app/private/pages/project/event-logs/event-logs.component.ts

```diff
@@ -195,16 +195,23 @@ export class EventLogsComponent implements OnInit {
 		this.enableTailMode = tailModeConfig;
 		localStorage.setItem('EVENT_LOGS_TAIL_MODE', JSON.stringify(tailModeConfig));
 		const data = this.getFiltersFromURL();
-		if (tailModeConfig) this.getEventLogs({ ...data, showLoader: true }).then(() => this.getEventsAtInterval({ ...data }));
+		if (tailModeConfig) this.startTailing();
 		else {
 			clearInterval(this.getEventsInterval);
 			this.getEventLogs({ ...data, showLoader: true });
```

Comment:
We should also add a 3s wait here before calling this guy, so it is uniform.

If you'll have to write it in multiple places, you can as well make it a function.